ignite/cmd/scaffold.go

```diff
@@ -5,12 +5,13 @@ import (
 	"fmt"
 
 	"github.com/AlecAivazis/survey/v2"
+	"github.com/spf13/cobra"
```

Comment:
Do you use any tool for imports formatting or you arrange them manually?

Reply:
we are using `goimports`. you can run with `make format`